### PR TITLE
Writing to the cancel code TABLE in our database in our load script.

### PR DESCRIPTION
**Pull request by HarryEP**
I have made 3 functions:
get_connection() which allows a connection to the database
get_cancel_code_csv_data() which gets the csv data that I manually made to put into a dataframe.
write_cancel_codes() which writes all the cancel code data directly to the database from the dataframe.

I am wanting to apply this to double check it's not accidentally missing the first line out or something like that.

### CODE REVIEW

File: load.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+'''Uploads data to the database'''
+import os
+import pandas as pd
+import psycopg2
+from psycopg2.extras import RealDictCursor
+from psycopg2.extensions import connection
+from dotenv import load_dotenv
+
+CODES_CSV = "cancel_codes.csv"
+
```
Comment (NgawangT) [inline load.py:10]: good use of constant for file.

File: load.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+def write_cancel_codes(conn: connection, codes_df: pd.DataFrame):
+    '''writes the cancel codes from the dataframe to the database'''
+    records = codes_df.to_records(index=False)
+    with conn.cursor() as cur:
+        sql_query = '''
+            INSERT INTO cancel_code (code, reason, abbreviation)
+            VALUES (%s, %s, %s)'''
+        cur.executemany(sql_query, records)
+
```
Comment (NgawangT) [inline load.py:20]: good use of with so the resources are closed and placeholder to prevent sql injection.

File: load.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    conn.commit()
+
+
+def get_cancel_code_csv_data() -> pd.DataFrame:
```
Comment (NgawangT) [inline load.py:24]: not sure if this needs to be a function as the line 26,  can be under if name == main?? maybe

File: load.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+import psycopg2
+from psycopg2.extras import RealDictCursor
+from psycopg2.extensions import connection
+from dotenv import load_dotenv
+
+CODES_CSV = "cancel_codes.csv"
+
+
+def write_cancel_codes(conn: connection, codes_df: pd.DataFrame):
+    '''writes the cancel codes from the dataframe to the database'''
```
Comment (LisheinLWM) [inline load.py:13]: Nice type hints and doc strings

**Review by NgawangT — APPROVED** [review]

**Pull request merged**